etna/models/nn/mlp.py

```diff
@@ -193,11 +256,13 @@ def __init__(
         Parameters
         ----------
         input_size:
-            size of the input feature space: target plus extra features
+            size of the input numeric feature space
```

Comment:
Why are you removing clarification about the target?

Reply:
I think it was a mistake. MLP doesn't get target as a feature

Reply:
Should we add that target doesn't count?
As I understand all these features should be regressors?

Reply:
* Let's do this
* Yes